frontend/src/components/commons/SideNavigation.jsx

```diff
@@ -11,8 +11,39 @@ class SideNavigation extends Component {
     currentNode: this.props.currentNode,
     currentBody: this.props.progressSpecs[this.props.currentNode].body
   };
+
+  changeNode(id) {
+    this.setState({ currentNode: id, currentBody: this.props.progressSpecs[id].body });
```

Comment:
some tests similar to these would be useful in this component: https://github.com/code-for-canada/project-thundercat/blob/master/frontend/src/tests/components/commons/TabNavigation.test.js

Reply:
added in e83657b